### DISCUSSION TITLE
Are there any project instances built using Vuepress-next

### DISCUSSION THREAD

**Discussion by F-one-1** [vuepress/core#496] [Q&A] [answered]
Are there any project instances built using Vuepress-next? At present I have some mistakes in the configuration process, want to learn from. Put a URL if you have one

**Comment by Joelius300** [comment]
I'm using vuepress@next for [Joelius300/werewolf-guide](https://github.com/Joelius300/werewolf-guide) and it's been amazing to work with!

**Reply by F-one-1** [reply]
thanks ,I meet the question is the he following error occurred when I tried to introduce a JSX component into Vuepress-next. I have no problem running vue3 project using yarn Serve. There was a problem using yarn Docs :dev.
[disscustion](https://github.com/vuepress/vuepress-next/discussions/493)

**Comment by Mister-Hope** [comment]
Here is a 120k+ lines project (39k+ lines of code): https://github.com/vuepress-theme-hope/vuepress-theme-hope

**Comment by Mister-Hope** [comment, marked as answer]
See [awesome-vuepress/v2](https://github.com/vuepress/awesome-vuepress/blob/main/v2.md)
